lib/graphs/abstract_digraph.nit

```diff
@@ -0,0 +1,583 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
+# Currently, there are two classes
+module abstract_digraph
+
+# Interface for digraphs
+abstract class AbstractDigraph[V]
```

Comment:
I suggest to remove the `Abstract` part of the name as it conveys more noise that information.